src/bundle.ts

```diff
@@ -67,20 +62,24 @@ export class BuildBundler extends Transform {
     const forkedAnalyzer = analyzer ? analyzer._fork({urlLoader}) :
                                       buildAnalyzer.analyzer._fork({urlLoader});
 
+    strategy = strategy ||
```

Comment:
Oh, I didn't notice that this was just moving this code here from `_generateBundleManifest`. LGTM